source/2020-08-25-accessibility-working-group-update.md

```diff
@@ -0,0 +1,73 @@
+
+The Strike Team met weekly between [March 25](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-03/march-25.md) and [June 24](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-06/june-24.md). We even managed to celebrate [Global Accessibility Awareness Day](https://blog.emberjs.com/2020/05/21/global-accessibility-awareness-day.html)!
+
+READMORE
+
+There were five items on the list: 
+
+1. Application Language Support
+2. Label/Input Support
+3. Page Title Support
+4. Support for `...attributes`
+5. Accessible Routing Support
+
+## Active Work 
+
+### Application Language Support
+
+<!-- alex ignore easy -->
+
+The goal of application language support was to provide a mechanism for new Ember apps to pass [WCAG Success Criteria 3.1.1 - Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page). To meet that goal, [Joseph Sumner](https://github.com/josephdsumner), [Jamie White](https://github.com/jgwhite), [Ava Wroten](https://github.com/hergaiety) and [Melanie Sumner](https://github.com/melsumner) authored [RFC 635](https://emberjs.github.io/rfcs/0635-ember-new-lang.html), which proposed a `--lang` flag for `ember-cli`. This flag would enable developers to declare the application's language while they were creating the app, allowing them to more easily meet this particular success criteria. This RFC was merged on June 12th; the feature itself, authored by [Joseph Sumner](https://github.com/josephdsumner) with help from [Rob Jackson](https://github.com/rwjblue), was merged on August 24th.
+
+To add another layer of support and enhance the overall developer experience, the team also decided that it would be useful to have an interactive way to create new Ember apps. After some discussion, [Melanie Sumner](https://github.com/melsumner) authored [RFC 638](https://emberjs.github.io/rfcs/0638-interactive-app-creation.html), and that RFC was accepted and merged on August 10th. Implementation is still pending.
+
+Additional support came through [updating the Ember Guides](https://guides.emberjs.com/release/accessibility/application-considerations/#toc_lang-attribute), with both prose and art, thanks to team members [Ava Wroten](https://github.com/hergaiety) and [Lenora Porter](https://github.com/lenoraporter). [Melanie Sumner](https://github.com/melsumner) also [added a new rule](https://github.com/ember-template-lint/ember-template-lint/pull/1386) to the [Ember Template Lint addon](https://github.com/ember-template-lint/ember-template-lint) called `require-lang`, which checks to make sure that the `lang` attribute is present on the `<html>` attribute. This rule will be effective in v3.0 of the addon, thanks to [PR 1232](https://github.com/ember-template-lint/ember-template-lint/pull/1232) by [lifeart](https://github.com/lifeart), which adds the ability to lint `.html` files.   
```

Comment:
I suggested the following:

- updated the first part of the 1st sentence
- broke the 2nd sentence into two
- removed the use of `which` in 3rd.

```suggestion
Additional support came from [update to the Ember Guides](https://guides.emberjs.com/release/accessibility/application-considerations/#toc_lang-attribute), in both prose and art, thanks to team members [Ava Wroten](https://github.com/hergaiety) and [Lenora Porter](https://github.com/lenoraporter). [Melanie Sumner](https://github.com/melsumner) also [added a new rule](https://github.com/ember-template-lint/ember-template-lint/pull/1386) to the [Ember Template Lint addon](https://github.com/ember-template-lint/ember-template-lint) called `require-lang`. It checks to make sure that the `lang` attribute is present on the `<html>` attribute. This rule will be effective in v3.0 of the addon, thanks to [Alex Kanunnikov](https://github.com/lifeart) adding the ability to lint `.html` files in [PR 1232](https://github.com/ember-template-lint/ember-template-lint/pull/1232).
```